index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="style.css" />
+    <title>Amina Afzal</title>
+  </head>
+  <body>
+    <h1>Amina Afzal</h1>
+    <div class="block">
+      <section class="block_box">
```

Comment:
Minor point, but it's best practice to stick to kebab-case for CSS, rather than snake_case.

```suggestion
      <section class="block-box">
```